docs/buildRelease.md

```diff
@@ -0,0 +1,53 @@
+# Auro build systems
+The goal of this document is to roughly outline every step of the build + release process in any given Auro component.
+
+## Local development
+When developing and/or working on an existing Auro component, a developer will initially -
+
+1. git clone a repository
+1. Run `nvm use 20` && `npm install` (nvm step optional)
+    1. As a post-install hook, git-hook tool husky installs scripts to the .git directory. More on this in the commit section!
+1. Checkout a new feature branch off main (some branches still use master)
+1. Run `npm run build` - this is a macro to build sass files and generate TypeScript declarations
+    1. This is inconsistent? Some have `build:version` and some do not
```

Comment:
Only components that are dependent on other auro components need `build:version`. But I think all components should have this as part of the build process and we should make that `build:version` script dynamic and able to be configured from one repo to the next, or smart enough to auto figure out what auro-components the repo is dependent on.